kubernetes/table_kubernetes_pod_disruption_budget.go

```diff
@@ -0,0 +1,151 @@
+package kubernetes
+
+import (
+	"context"
+
+	v1beta1 "k8s.io/api/policy/v1beta1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+func tableKubernetesPdb(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "kubernetes_pod_disruption_budget",
+		Description: "A Pod Disruption Budget limits the number of Pods of a replicated application that are down simultaneously from voluntary disruptions.",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"name", "namespace"}),
+			Hydrate:    getPdb,
+		},
+		List: &plugin.ListConfig{
+			Hydrate:    listPdbs,
+			KeyColumns: getCommonOptionalKeyQuals(),
```

Comment:
Optional key columns need to handle in the list call. Please refer `kubernetes_replicaset` or `kubernetes_pod` tables